en/docs/chapter_tree/binary_search_tree.md

```diff
@@ -59,13 +59,13 @@ Similar to searching for a node, inserting a node uses $O(\log n)$ time.
 
 ### Removing a node
 
-First, find the target node in the binary tree, then remove it. Similar to inserting a node, we need to ensure that after the removal operation is completed, the property of the binary search tree "left subtree < root node < right subtree" is still satisfied. Therefore, based on the number of child nodes of the target node, we divide it into 0, 1, and 2 cases, performing the corresponding node removal operations.
+First, find the target node in the binary tree, then remove it. Similar to inserting a node, we need to ensure that after the removal operation is completed, the property of the binary search tree "left subtree < root node < right subtree" is still satisfied. Therefore, based on the number of child nodes of the target node, we divide it into three cases: 0, 1, and 2. And perform the corresponding node removal operations.
 
-As shown in the figure below, when the degree of the node to be removed is $0$, it means the node is a leaf node, and it can be directly removed.
+As shown in the figure below, when the degree of the node to be removed is $0$, it means the node is a leaf node and can be directly removed.
 
 ![Removing a node in a binary search tree (degree 0)](binary_search_tree.assets/bst_remove_case1.png)
 
-As shown in the figure below, when the degree of the node to be removed is $1$, replacing the node to be removed with its child node is sufficient.
+As shown in the figure below, when the degree of the node to be removed is $1$, replacing this node with its child node is sufficient.
```

Comment:
i think the original:
"As shown in the figure below, when the degree of the node to be removed is $1$, replacing the node to be removed with its child node is sufficient."
is better